examples/sse_streaming/limits.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- namespace: domain_a
```

Comment:
should this be `domain-a` to stay consistent with the scope set [here](https://github.com/Kuadrant/wasm-shim/pull/215/files)